azure-pipelines.yml

```diff
@@ -820,6 +820,8 @@ jobs:
   - bash: |
       set -ev
       . env/bin/activate
+
+      pip install --force -q git+https://github.com/Azure/azure-cli-dev-tools.git@master
```

Comment:
Aren't we using the PyPI version of `azdev`?

Reply:
Yeah. This PR just for verifying the result, the feature is still in the master branch of azdev. I will release a newer azdev once https://github.com/Azure/azure-cli/pull/14402 is megred.